src/scrcmd.c

```diff
@@ -1309,6 +1310,7 @@ bool8 ScrCmd_release(struct ScriptContext *ctx)
     ObjectEventClearHeldMovementIfFinished(&gObjectEvents[playerObjectId]);
     ScriptMovement_UnfreezeObjectEvents();
     UnfreezeObjectEvents();
+	gMsgBoxIsCancelable = FALSE;
```

Comment:
spaces instead of tabs

Reply:
[`f0bd3c5` (#5044)](https://github.com/rh-hideout/pokeemerald-expansion/pull/5044/commits/f0bd3c57daa2a32eca39e01b8f10c92d85d8e4bf)